### DISCUSSION TITLE
Disable / Unload a template or template part

### DISCUSSION THREAD

**Discussion by spidrwebuk** [WordPress/gutenberg#46961] [General]
Hi all, hope you're all good!

Is there a way to **unload** or **disable** a template or template part in Site Editor?

I have a block theme that works well for use on multiple sites, however there are cases where I do not want to show the WooCommerce templates where WooCommerce isn't installed.

Thoughts and Feedback would be appreciated.

**Comment by talldan** [comment]
Is it a code solution you're looking for?

I believe there's a filter for block templates - https://developer.wordpress.org/reference/hooks/get_block_templates/

**Reply by spidrwebuk** [reply]
Aha... I will look at that, Thank you!

If it does the trick or not, I'll report back here either way!

:)
